### ISSUE TITLE
Error: Error Attempting to read Image

### ISSUE THREAD

**Issue by Allen1801** [naptha/tesseract.js#978]
So, I am creating a TauriV2 + Leptos project that involves tesseract.js CDN. I can successfully upload and read the image inside the project folder but the issue is when I read an image outside the project folder, I get an error saying (Error: Error attempting to read image.")

I want to know if it is possible for tesseract.js to access images outside the project folder 

![image](https://github.com/user-attachments/assets/3a8e8a10-53f4-40e7-9583-552da08d4fda)

**Comment by Balearica** [comment]
Can you confirm whether you are able to access these files in another library/function (not Tesseract.js)?  If not, then I think this is a Tauri question rather than a Tesseract.js question.  Frameworks for building desktop applications such as Electron and Tauri do limit what filesystem-related features are enabled by default for security reasons.  Current best practices for accepting input files should be discussed somewhere in the documentation for those programs.

**Comment by Allen1801** [comment]
Yes, I can access these files using the tesseract library, but I opted to use a cdn because I need to use the app on mobile

**Comment by Balearica** [comment]
> Yes, I can access these files using the tesseract library, but I opted to use a cdn because I need to use the app on mobile

It is unclear to me what this means.  Are you claiming that the browser version of Tesseract.js works if you host all of the files locally but does not work when the Tesseract.js files are hosted on a CDN?  

If you want assistance, the best thing you can do is create a minimal reproducible example that can be cloned and easily run to reproduce the issue.  Without that, all we can do is guess as to what is going on.
